cabot/settings.py

```diff
@@ -13,7 +13,7 @@
 settings_dir = os.path.dirname(__file__)
 PROJECT_ROOT = os.path.abspath(settings_dir)
 
-TEMPLATE_DEBUG = DEBUG = os.environ.get("DEBUG", False)
+DEBUG = os.environ.get("DEBUG", False)
```

Comment:
Does this even work? Won't the environment variable always be a string ("True", "False")? (Evan saw this somewhere else)

Reply:
Yeah, if the `DEBUG` env var is set to a non-empty string, it's always considered true. I think it's always worked this way (and as you said, several other vars work the same).